src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -672,11 +613,13 @@ public void testCheckAddsNewResourceLocationButKeepsSameCheckerInstance() throws
         // Cache should not be reused.
```

Comment:
Can't comment on method name, but same comment as others, add cache somewhere to method name.